build.sbt

```diff
@@ -2,21 +2,24 @@ organization := "org.virtuslab"
 
 name := "play-slick-configuration"
 
-version := "2.2.1-SNAPSHOT"
+version := "2.3.0"
```

Comment:
Please make it 2.3.0-SNAPSHOT, I will bump version during release process :)

Reply:
Of course, changed